src/components/calculator.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './calculator.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
```

Comment:
`Calculator.js` component is class-based 😎

Reply:
done

Reply:
Just pointing out the key requirements you did well 😉